### PR TITLE
Updated Puppetfile for ASDPLNG-46

### PR DESCRIPTION
**Pull request by cpitcel**
added include for additional packages

testing, to add tree to additional packages

removing testing. It worked!

lets try this again :(

so it did work. I am smart

change to Puppetfile to update the additional packages version

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ mod 'herculesteam/augeasproviders_ssh', commit: 'e4eee3726d0472cba1d2d66a2d09031
 mod 'inkblot/ipcalc', '2.2.0'
 # mod 'MiamiOH-pam_access', '1.0.2'
 mod 'ncsa/pam_access', tag: 'v1.0.3', git: 'https://github.com/ncsa/puppet-pam_access'
-mod 'ncsa/profile_additional_packages', tag: 'v0.1.0', git: 'https://github.com/ncsa/puppet-profile_additional_packages'
+mod 'ncsa/profile_additional_packages', branch: 'v0.2.0', git: 'https://github.com/ncsa/puppet-profile_additional_packages'
```
Comment (andylytical) [inline Puppetfile:13]: This should be "tag"

**Review by andylytical — CHANGES_REQUESTED** [review]
Puppetfile should use "tag" (not "branch")

File: Puppetfile (hunk 1 above)
Comment (cpitcel) [reply Puppetfile]: Changed

**Review by andylytical — APPROVED** [review]
Looks good.

**Comment by cpitcel** [comment]
Merge'n

**Pull request merged**